packages/sarif-to-markdown/src/sarif-to-markdown.ts

```diff
@@ -121,7 +126,7 @@ export const sarifToMarkdown = (options: sarifFormatterOptions): ((sarifLog: Log
             /**
              * # Rule Info
              */
-            const ruleInfo = escapeMarkdown`
+            let ruleInfo = escapeMarkdown`
```

Comment:
I think that we can extract the task as a function like `createRuleInfo`.
If we do the refactoring, we can use `const`.
```js
const ruleInfo = simpleMode ? "" : createRuleInfo({ ... })
```